language-server/src/dotty/tools/languageserver/DottyLanguageServer.scala

```diff
@@ -795,12 +796,17 @@ object DottyLanguageServer {
 
     val label = sym.name.show
     val item = new lsp4j.CompletionItem(label)
-    item.setDetail(sym.info.widenTermRefExpr.show)
+    val detail = if (sym.isType) sym.showFullName else sym.info.widenTermRefExpr.show
```

Comment:
Idea: instead of just showing the full name we could show the kind of the symbol too, e.g. "class a.b.c.Foo", "val x", ... and in the case where there's more than one symbol with the same name, we could show both at the same time "class and object a.b.c.Foo", ... We can reuse the code used in the REPL when printing a declaration.